onnxruntime/test/contrib_ops/function_ops_test.cc

```diff
@@ -108,5 +108,26 @@ TEST_F(ContribFunExpansionTest, FastGeluWithoutBias) {
   CheckFastGelu<MLFloat16, false>(false);
 }
 
+template <typename T, bool RunTest = true>
+void CheckBernoulli() {
+  FunctionTestCase testCase("Bernoulli", kMSDomain);
+  std::vector<int64_t> shape{8, 12};
+
+  testCase.AddInput<T, RunTest>("x", shape);
+  const int64_t seed = 42;
+  testCase.AddAttribute("seed", seed);
+  testCase.AddOutput("y");
```

Comment:
Would be good to also test the case where attribute dtype is specified (as well as the current case where it is not specified).

Reply:
Similarly, check the case where seed attribute is not specified.

Reply:
> Would be good to also test the case where attribute dtype is specified (as well as the current case where it is not specified).

Thanks, added in latest commit.

Reply:
> Similarly, check the case where seed attribute is not specified.

I have tested this situation, every test will run 128 times. If not specified seed, it will be failed at one random time.